packages/docs/pages/spacing.tsx

```diff
@@ -1,9 +1,25 @@
-import { Box, Button, Flex, H1, Panel, Text } from '@bigcommerce/big-design';
+import { Box, Button, Flex, H1, Panel, Table, Text } from '@bigcommerce/big-design';
 import React, { Fragment, useContext } from 'react';
 import styled, { ThemeContext } from 'styled-components';
 
 import { Code, CodePreview, ContentRoutingTabs, List, NextLink } from '../components';
 
+// If your theme exports a SPACING_KEYS array, prefer importing it:
+// import { SPACING_KEYS } from '@bigcommerce/big-design-theme';
+
+// Keep this local list in docs to control order & avoid negatives/none in the visual demo.
+// (If you later export SPACING_KEYS from the theme, swap to that single source of truth.)
+const POSITIVE_KEYS = [
+  'xxSmall',
+  'xSmall',
+  'small',
+  'medium',
+  'large',
+  'xLarge',
+  'xxLarge',
+  'xxxLarge',
+];
```

Comment:
Can we try an just filter this out via the object so if we add new spacing values it automatically reflects?
```ts
Object.keys(theme.spacing).filter(key => !key.endsWith("N"))...
```